cassandra.go

```diff
@@ -75,6 +75,20 @@ func Open(rawurl string) (driver.Driver, error) {
 		}
 
 	}
+	// handle ssl option
+	if len(u.Query().Get("sslmode")) > 0 {
```

Comment:
For the sake of code consistency, shouldn't we have 

```go
if sslMode, ok := u.Query()["sslmode"]; ok {
```

instead? If this syntax is more idiomatic, then would you mind changing the rest of the file accordingly?
The next line would not be needed then.

Reply:
This code doesn't make sense to me: 

```
if sslmode := u.Query().Get("sslmode"); sslmode != "" {
	if sslmode != "disable" {
```

Why don't you use instead: `if sslmode := u.Query().Get("sslmode"); sslmode != "disable" {``

Reply:
`u.Query().Get("sslmode")` could return zero-value if we don't assign `sslmode`.
It will enter the block and init `cluster.SslOpts`.
how about :
```
if sslmode := u.Query().Get("sslmode"); sslmode != "" && sslmode != "disable" {
```

Reply:
ha ok. In this case, shouldn't it be `sslmode != "" || sslmode != "disable" instead?
Because `sslmode != "" && sslmode != "disable"` is pretty much only `sslmode != "disable"`

Reply:
`sslmode != "" || sslmode != "disable"`, In this condition, if will init `cluster.SslOpts`, if the user doesn't set `sslmode`.
But I think, we shouldn't init `cluster.SslOpts`.
Because of the user doesn't specific `sslmodel` in the URL.

The following case we should init `cluster.SslOpts`.
1. user set `sslmode`, this mean u.Query().Get("sslmode") is not empty string. 
condition: `sslmode != ""`.
2. `sslmode` is not disable.
condition: `sslmode != "disable"`

Combine 1. and 2., the condition will be `sslmode!= "" && sslmode != "disable"`.

Reply:
Ok, thanks for sharing. I get your point.